keyboards/aplusx_pulsar/xboard_qs/xboard_qs.c

```diff
@@ -0,0 +1,255 @@
+/* Copyright 2024 Aplusx Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#include "xboard_qs.h"
+
+static bool     INIT3S_on = false;
+static bool     INIT3S_led_on = false;
+static uint16_t INIT3S_timer;           // for custom key, INIT_3S
+
+static bool     kvm_pc_sel      = false;
+static uint16_t kvm_timer;
+static bool     kvm_sel_on      = false;
+
+static bool isMacMode = false;
+static uint8_t Status_LED_Bright;
+#define kvm_deadtime    100
+
+typedef union {
+    uint32_t raw;
+    struct {
+        bool    eeprom_kvm_pc_sel :1;
+        uint8_t eeprom_Status_LED_Bright;       //
+    };
+} user_config_t;
+
+user_config_t user_config;
+
+void kvm_switch(bool pc_num){
+    writePinHigh(GPIO_KM_OE);
+    writePinLow(GPIO_KM_PWEN);
+    if (!pc_num) writePinLow(GPIO_KM_SEL);
+    else   writePinHigh(GPIO_KM_SEL);
+    writePinLow(GPIO_KM_OE);
```

Comment:
pin control names have changed recently.
```suggestion
    gpio_write_pin_high(GPIO_KM_OE);
    gpio_write_pin_low(GPIO_KM_PWEN);
    if (!pc_num) gpio_write_pin_low(GPIO_KM_SEL);
    else   gpio_write_pin_high(GPIO_KM_SEL);
    gpio_write_pin_low(GPIO_KM_OE);
```

Reply:
I got it. thank you